tests/src/Functional/FunctionalTestCase.php

```diff
@@ -51,8 +51,11 @@ protected function configureOptions()
 
     protected function setup()
     {
+        $testsDir = realpath(__DIR__ . '/../..');
+
         $class = new ReflectionClass($this);
-        $this->outputDir = 'tests/generated/' . $class->getShortName();
+        $this->outputDir = $testsDir . '/generated/' . $class->getShortName();
+        $this->fixtureDir = $testsDir . '/fixtures/wsdl';
```

Comment:
If we set `fixtureDir` during setup the please remove [the default value](https://github.com/yethee/wsdl2phpgenerator/blob/inline-type/tests/src/Functional/FunctionalTestCase.php#L32).